andela/ah-frontend-taces
# #159951838 Users should be able to highlight and comment on any text in an article

#### What does this PR do?
This PR adds functionality to allow users to highlight and comment on text within an article body.
#### Test Coverage
<img width="1440" alt="screen shot 2018-09-20 at 19 11 04" src="https://user-images.githubusercontent.com/28563179/45833137-92813a80-bd0c-11e8-9824-b99fabf6178c.png">

#### How should this be manually tested?
Select any text in the article and a pop-up box will appear in the right bottom corner where you can comment on the text you have selected. To view all responses made on text within that article, click on "VIEW ALL RESPONSES" button in the pop-up. To close the pop-up, double-click within the article.
#### What are the relevant pivotal tracker stories?
[#159951838](https://www.pivotaltracker.com/story/show/159951838)
#### Screenshots
<img width="1440" alt="screen shot 2018-09-20 at 19 34 48" src="https://user-images.githubusercontent.com/28563179/45833043-4d5d0880-bd0c-11e8-8cee-bee68b5d476b.png">
